src/selectors/entities/channels.test.js

```diff
@@ -9,6 +9,884 @@ import {sortChannelsByDisplayName, getDirectChannelName} from 'utils/channel_uti
+    profiles[user2.id] = user2;
+    profiles[user3.id] = user3;
+
+    const channel1 = TestHelper.fakeChannelWithId(team1.id);
+    channel1.display_name = 'Channel Name';
+    const channel2 = TestHelper.fakeChannelWithId(team2.id);
+    channel2.display_name = 'Channel Name';
+    const channel3 = TestHelper.fakeChannelWithId(team1.id);
+    channel3.display_name = 'Channel Name';
+    const channel4 = TestHelper.fakeChannelWithId('');
+    channel4.display_name = 'Channel Name';
+    channel4.type = General.DM_CHANNEL;
+    channel4.name = getDirectChannelName(user.id, user2.id);
+    const channel5 = TestHelper.fakeChannelWithId('');
+    channel5.display_name = [user.username, user2.username, user3.username].join(', ');
+    channel5.type = General.GM_CHANNEL;
+    channel5.name = '';
+
+    const channels = {};
+    channels[channel1.id] = channel1;
+    channels[channel2.id] = channel2;
+    channels[channel3.id] = channel3;
+    channels[channel4.id] = channel4;
+    channels[channel5.id] = channel5;
+
+    const channelsInTeam = {};
+    channelsInTeam[team1.id] = [channel1.id, channel3.id];
+    channelsInTeam[team2.id] = [channel2.id];
+    channelsInTeam[''] = [channel4.id, channel5.id];
+
+    const myMembers = {};
+    myMembers[channel1.id] = {};
+    myMembers[channel3.id] = {};
+    myMembers[channel4.id] = {};
+    myMembers[channel5.id] = {};
+
+
+    const testState = deepFreezeAndThrowOnMutation({
+        entities: {
+            users: {
+                currentUserId: user.id,
+                profiles,
+                statuses: {},
+                profilesInChannel: {
+                    [channel4.id]: new Set([user.id, user2.id]),
+                    [channel5.id]: new Set([user.id, user2.id, user3.id]),
+                }
+            },
+            teams: {
+                currentTeamId: team1.id,
+            },
+            channels: {
+                channels,
+                channelsInTeam,
+                myMembers,
+            },
+            preferences: {
+                myPreferences: {}
+            },
+            general: {
+                config: {}
+            }
+        },
+    });
+
+    it('get my channels in current team and DMs', () => {
+        const channelsInCurrentTeam = [channel1, channel3].sort(sortChannelsByDisplayName.bind(null, []));
+        assert.deepEqual(Selectors.getMyChannels(testState), [
+            ...channelsInCurrentTeam,
+            {...channel4, display_name: user2.username, status: 'offline', teammate_id: user2.id},
+            {...channel5, display_name: [user2.username, user3.username].sort(sortUsernames).join(', ')}
+        ]);
+    });
+});
+
+describe('Selectors.Channels.getMembersInCurrentChannel', () => {
+    const channel1 = TestHelper.fakeChannelWithId('');
+
+    const user = TestHelper.fakeUserWithId();
+    const user2 = TestHelper.fakeUserWithId();
+    const user3 = TestHelper.fakeUserWithId();
+
+    const membersInChannel = {};
```

Comment:
Using an object literal for this whole definition as I mentioned above would definitely help readability here